### ISSUE TITLE
DAPT Data Model: Harmonize use of styles

### ISSUE THREAD

**Issue by andreastai** [w3c/dapt#15]
Styles are used for characters, texts and contextual texts. From my reading, they are essentially the same property for all three "contexts" where there are used but they have a different name in each context (`Style`, `Text Styles` and `Contextual Text Styles`). This could be represented as one property (e.g. `Style`) that has a cardinality of `1..*` and references a `Style Object` (with style feature e.g. `color`). The association is implied by the object it belongs to.

**Comment by nigelmegitt** [comment]
I propose to add a section on Styling, with two sub-sections, one for visual styling and the other for audio styling. This needs to describe the mechanisms and optionality for each, and how styles can be applied to script events, text and characters.

Specifically, all visual styling is optional, but audio styling is required in order to define mixing instructions.

We can remove Style objects from the data model.

**Comment by nigelmegitt** [comment]
This may be resolved by #76 which defines a Character Style object - see what you think @andreastai . My doubt here is that not all styles are defined by reference to `<style>` objects: inline styling is also permitted, but that's not really associated with a Style object from a data modelling perspective.

**Comment by nigelmegitt** [comment]
Possibly enough changes were made in #76 to close this?

**Comment by cconcolato** [comment]
I suggest closing this issue as I think the initial concern has been addressed.

**Comment by nigelmegitt** [comment]
Once #86 is done I propose to close this.

**Comment by andreastai** [comment]
Thank you for addressing the original issue. However, I have another concern regarding the concept of character styles in the specification. I am having trouble seeing the benefits of this feature and will open a separate issue to discuss it further

**Comment by andreastai** [comment]
@nigelmegitt I think the tag "done awaiting confirmation" can be removed
